.travis.yml

```diff
@@ -1,35 +1,44 @@
-language: cpp
+language: python
 
-os: 
-    - osx
-    - linux 
-
-env:
-    matrix:
-        - TEST_SUITE="bin/py.test -n 2 -vvs --ignore=tests/scancode --ignore=tests/extractcode --ignore=tests/licensedcode --ignore=tests/cluecode --ignore=tests/packagedcode"
-        - TEST_SUITE="bin/py.test -n 2 -vvs tests/scancode"
-        - TEST_SUITE="bin/py.test -n 2 -vvs tests/extractcode"
-        - TEST_SUITE="bin/py.test -n 2 -s tests/licensedcode"
-        - TEST_SUITE="bin/py.test -n 2 -s tests/cluecode"
-        - TEST_SUITE="bin/py.test -n 2 -s tests/packagedcode"
-        - TEST_SUITE="./etc/release/release.sh"
+python:
+  - 2.7.12
+  - 2.7.13
```

Comment:
Made two versions just because I could. That is indeed making the build twice as long, will reduce to `2.7.13`